google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineCloudConfigurable.java

```diff
@@ -82,24 +75,11 @@ public JComponent createComponent() {
 
   @Override
   public boolean isModified() {
-    // Forces a modify check so the user is unable to save an invalid Cloud SDK configuration from
-    // Other Settings, on the Clouds menu.
-    return cloudSdkPanel.isModified()
-        || !CloudSdkService.getInstance().isValidCloudSdk(cloudSdkPanel.getCloudSdkDirectoryText());
+    return cloudSdkPanel.isModified();
   }
 
-  /**
-   * Users shouldn't be able to save a cloud configuration without a valid Cloud SDK configured.
-   */
   @Override
-  public void apply() throws ConfigurationException {
-    String message = cloudSdkPanel.buildSdkMessage(cloudSdkPanel.getCloudSdkDirectoryText(),
-        false /*htmlEnabled*/);
-
-    if (!StringUtil.isEmpty(message)) {
-      throw new RuntimeConfigurationError(message);
```

Comment:
Here's the actual fix. The rest is some clean-up in the CloudSdkPanel and the conversion of the test to use the new rule.